index.bs

```diff
@@ -11115,12 +11118,20 @@ in the algorithm of <a href="#rendering-a-graph">rendering a graph</a>.
 	finished.
 </div>
 
-The audio callback is also queued as a task in the
-<a href="#control-message-queue">control message queue</a>. The UA MUST perform
+The audio callback is also queued as a task in the <a
+	href="#control-message-queue">control message queue</a>. The UA MUST perform
```

Comment:
nit: do we need to indent here?

Reply:
No idea. I'll check locally by running `bikeshed` and merge as is if needed, or fixed if not. Thanks for the quick re-review!

Reply:
👍

Reply:
(`bikeshed` accepts it).